_posts/2024-04-16-pyos-open-education-resources-announcement.md

```diff
@@ -0,0 +1,99 @@
+
+## Enroll in pyOpenSci’s upcoming workshop: From Python Code to Module
+<figure>
+    <a href="/images/blog/2024/april/pyos-code-to-mod.png">
+    <img src="/images/blog/2024/april/pyos-code-to-mod.png" style="max-width:100%" alt="A line art robot standing in a field of purple flowers. The text reads 'From Python Code to Module, a live, online workshop with pyOpenSci, Thursday, April 25th 2024'.">
+    </a>
+    <figcaption>
+      Tickets for our "From Python Code to Module" workshop are selling fast - get yours today!
+    </figcaption>
+</figure>
+If you’re interested in participating in our first paid, online, real-time training, sign up for our upcoming workshop: [“From Python Code to Module”](https://www.eventbrite.com/e/879586546037?aff=oddtdtcreator). This three hour course is intended for individuals who have experience writing Python code and Python functions, and will be taught by pyOpenSci’s Executive Director and founder, [Leah Wasser](https://github.com/lwasser). Leah has over 20 years of experience teaching data-intensive science with an emphasis helping scientists work through the pain points of working with different types of data, and puts an incredible amount of care and attention into ensuring each learner is successful in their educational goals. This is definitely a workshop you don’t want to miss!
+
+In this workshop, Leah will cover:
+- How to identify and explain the use of the basic components of a Python package: (a specific directory structure, an __init__.py file, a pyroject.toml file and some code).
+- Creating a basic python package that allows you to install your code into a local Python environment.
+- Installing your package in editable mode into a Python environment.
+
+She will also briefly discuss how LLM’s can be used to support tasks such as documenting and formatting your code to improve usability and maintainability. While also considering the ethical and logistical challenges, pitfalls and concerns associated with using AI-based tools in software development.
+
+The course will take place on Thursday, April 25th, from 10AM–1PM Mountain time, and use the Spatial Chat platform. Tickets are $10, and can be purchased on [the workshop’s Eventbrite page](https://www.eventbrite.com/e/from-python-code-to-module-tickets-879586546037?aff=oddtdtcreator). The class size is capped at 35, and tickets are selling fast–we hope to see you there!
+
+## Connect with us!
+You can stay up-to-date with all things pyOpenSci by following us on [LinkedIn](https://www.linkedin.com/company/pyopensci) and [Fosstodon](https://fosstodon.org/@pyOpenSci), and you can connect with the broader pyOpenSci community on [our Discourse forum](https://pyopensci.discourse.group/). And if you’re interested in our weekly newsletter where we share news, blog posts, and monthly updates, [subscribe on LinkedIn](https://www.linkedin.com/newsletters/pyopensci-newsletter-7179551305344933888!
```

Comment:
looks like this link is missing a closing bracket and as such not rendering. 

You can see what the [page rendered looks like here](https://output.circle-artifacts.com/output/job/c1617bb4-475f-4608-b778-c03ff4ce4726/artifacts/0/_site/blog/pyos-oer-announcement.html)

the page can't be loaded in the direct CI link on the PR for some reason (likely because its dynamically generated?) but you can still get to it by clicking the "middle" circle CI link that says ci/circleci build. and then go to "artifacts" to find a listing of each individual page. from there you can then click on the blog post and view. 
 
<img width="1117" alt="Screenshot 2024-04-16 at 9 04 28 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/9515e841-bb06-432c-a24b-529de6229192">

<img width="947" alt="Screenshot 2024-04-16 at 9 03 54 AM" src="https://github.com/pyOpenSci/pyopensci.github.io/assets/7649194/f9eb96d7-8eb5-4f5c-ab8f-0f66ffe33eab">


```suggestion
You can stay up-to-date with all things pyOpenSci by following us on [LinkedIn](https://www.linkedin.com/company/pyopensci) and [Fosstodon](https://fosstodon.org/@pyOpenSci), and you can connect with the broader pyOpenSci community on [our Discourse forum](https://pyopensci.discourse.group/). And if you’re interested in our weekly newsletter where we share news, blog posts, and monthly updates, [subscribe on LinkedIn](https://www.linkedin.com/newsletters/pyopensci-newsletter-7179551305344933888)!
```

Reply:
can't believe I missed that one 🤦‍♀️

Reply:
no worries! It happens to me ALL of the time. I've literally linked to wrong images, forgot i left myself a comment that shouldn't be public, etc. This is all great! that is why reviews can be so nice :) fresh eyes find new things! great work knocking this out!